package.json

```diff
@@ -34,6 +35,7 @@
     "glob": "^7.1.2",
     "rimraf": "^2.6.2",
     "ts-node": "^3.3.0",
-    "typescript": "^3.1.3"
+    "typescript": "4.6.3",
```

Comment:
peerDependencies for typescript could be removed

Reply:
removed